python/build-wheel-windows.sh

```diff
@@ -133,8 +133,11 @@ build_wheel_windows() {
       fi
       # build ray wheel
       python setup.py --quiet bdist_wheel
+      # Pack any needed system dlls like msvcp140.dll
+      delvewheel repair dist\ray-*.whl
```

Comment:
Is this the only place wheels are built? I am not clear on the process used to upload wheels to PyPI

Reply:
@can-anyscale can you help here?